packages/server/src/profile/profile.controller.ts

```diff
@@ -333,4 +334,25 @@ export class ProfileController {
       );
     }
   }
+
+  // PAT TODO: Find better place for this route
```

Comment:
lol, yeah this is probably a fine spot to have the route

Reply:
I keep forgetting to ctrl+F for those before making the PR. My bad, hahaha